docs/config.rst

```diff
@@ -160,22 +160,6 @@ Global settings are defined under the ``tox`` section as:
     Name of the virtual environment used to create a source distribution from the
     source tree.
 
-.. conf:: interrupt_timeout ^ float ^ 0.3
```

Comment:
Why move these?

Reply:
This PR bundles up two commits. I can separate them out if needed, but it is explained in the earlier message:

> Move timeout config options into correct section
> 
> The interrupt_timeout and terminate_timeout options are set on the
> environment, not globally.

Reply:
(I'll update the PR message at least)